docs/source/el7toel8/envars.md

```diff
@@ -2,12 +2,45 @@
 
 Actors in the el7toel8 repository use environment variables specified below.
 All these envars use the suggested prefixes specified in
-[the best practices document](../best-practices.html#use-the-leapp-and-leapp-devel-prefixes-for-new-envars)
+[the best practices document](../best-practices#use-the-leapp-and-leapp-devel-prefixes-for-new-envars)
 for the leapp project to distinguish their purpose: *production* or *devel* use.
 
 If the argument for envars below is not specified, it is expected to set `0`
 (false) or `1` (true).
 
+## LEAPP_NO_RHSM
+
+Do not use Red Hat Subscription Management for the upgrade. Using it has the
+same effect as using the `--no-rhsm` leapp option.
+
+## LEAPP_OVL_SIZE
+
+TODO
```

Comment:
@pirat89, please help me here.

Reply:
```suggestion

In case any partitions use XFS without ftype attribute, we need to create for every such FS file of specific size, to by able to proceed the upgrade (skipping details what we do with such files). By default, the size is 2048 MB for every such file. In some cases the size has to be changed. If you meet such case, leapp usually informs you that the environment variable needs to be specified (e.g. inside the preupgrade report).
```

Reply:
Thanks. I've tweaked it a bit if you don't mind.